vllm/platforms/interface.py

```diff
@@ -23,6 +23,9 @@ def is_rocm(self) -> bool:
     def is_tpu(self) -> bool:
         return self._enum == PlatformEnum.TPU
 
+    def is_xla(self) -> bool:
+        return self.is_tpu()
+
```

Comment:
why do we need this? isn't `is_tpu()` enough?

Reply:
You're right. Neuron actually uses XLA, but it does not use `vllm/distributed`.

Reply:
Fixed!